tests/server_test.js

```diff
@@ -10,6 +10,53 @@ test('Testing empty root get request - / ', t => {
     .expect(200)
     .end(() => { 
       t.end();
-      process.exit(0);
     });
 });
+test('Testing the root /homePagePosts  ', t => {
+  const expectedThings = [
+    {
+      id: 1,
+      type: 'story',
+      title: 'My epic party',
+      content: 'Hey, all invited',
+      time_stamp: new Date('20017-12-16 06:00:00'),
+      tagids: 1
+    }
+  ];
+  request(app)
+    .get('/homePagePosts')
+    .expect('Content-Type', /json/)
+    .expect(200)
+    .end((err, res) => {
+      t.ok(res.text);
+      let actualThings = res.body.homePagePosts;
+      console.log(actualThings);
```

Comment:
please change this line to 
please remove the console.logs or tag them with TODO for later fixes